packages/node-sdk/paima-utils-backend/src/cde-access-internals.ts

```diff
@@ -158,6 +167,50 @@ export async function internalGetGenericDataBlockheightRange(
   }));
 }
 
+export async function internalGetErc1155TotalBalanceAllTokens(
+  readonlyDBConn: Pool,
+  cde_id: number,
+  wallet_address: string
+): Promise<bigint> {
+  const results = await cdeErc1155GetTotalBalanceAllTokens.run(
+    { cde_id, wallet_address },
+    readonlyDBConn
+  );
+  return BigInt(results[0].total ?? '0');
+}
```

Comment:
Isn't this dangerous as a utility function?

1. For ERC-1155 that are not for inverse projections, this utility function is never useful (since nobody else has erc1155 collections like us where all the tokens are the same)
2. For inverse projection use-cases, we need extra information to know which token ID is a valid mint and which one isn't

Reply:
Hmm, makes sense. Previously the Tarochi side was doing this query, but it never inserted invalid mints to begin with. Now Tarochi instead cross-references with its table of which mints were valid. I've gone ahead and trashed this query.